charts/alertmanager/values.yaml

```diff
@@ -41,7 +41,7 @@ priorityClassName: ""
 
 podSecurityContext:
   fsGroup: 65534
-dnsConfig: {}
+dnsConfig: []
```

Comment:
this does not look like a lst, as the first comment starts with: "# nameservers:"

Reply:
Yeah, not sure why I did this. Sorry, hostAliases is the only impacted service. I'll amend.